R/readGT3X.R

```diff
@@ -82,6 +95,9 @@ read.gt3x <- function(path, verbose = FALSE, asDataFrame = FALSE,
   path = unzip_zipped_gt3x(path, cleanup = cleanup)
```

Comment:
For consistency, could you use "<-" for assignment always? I believe that is the recommendation in R code.